src/stylesheets/components/_page-background.scss

```diff
@@ -1,25 +1,21 @@
 .page-background {
+  width: 100vw;
+  height: 100vh;
+
   &__image {
-    bottom: 0;
-    left: 0;
+    width: 100%;
+    height: 100%;
+
     overflow: auto;
-    position: absolute;
-    right: 0;
-    top: 0;
     background: {
-      color: $primary-color;
+      color: map-get($brand-black, 'primary');
       size: cover;
```

Comment:
I think we need `background-repeat:no-repeat;` also @rafayet-monon

Reply:
Fixed in [6ada2d2](https://github.com/rafayet-monon/surveyor/pull/13/commits/6ada2d22d37cb0fa39af1ba8e59a8880825ee44e)